src/soundio/sounddevicestatus.h

```diff
@@ -1,9 +1,9 @@
 #pragma once
 
 // Used for returning errors from sounddevice functions.
-enum SoundDeviceError {
-    SOUNDDEVICE_ERROR_ERR = -1,
-    SOUNDDEVICE_ERROR_OK = 0,
+enum SoundDeviceStatus {
```

Comment:
If you are already touching everything, we might as well make this an enum class IMO. 

```cpp
enum class SoundDeviceStatus {
    Error = -1,
    Ok = 0,
    ErrorDuplicateOutputChannel,
    ErrorExcessiveOutputChannel,
    ErrorExcessiveInputChannel,
    ErrorDeviceCount
};

Reply:
No prob, done.